lib/suse/connect/migration.rb

```diff
@@ -23,9 +23,14 @@ def rollback(client_params = {})
           client = Client.new(config)
           status = Status.new(config)
 
-          # FIXME: Sort products and ensure the base product is the first one in the list
-          status.installed_products.sort_by { |p| p.isbase ? 0 : 1 }.each do |product|
-            service = client.downgrade_product(product)
+          tree = client.show_product(Zypper.base_product)
```

Comment:
This calls the product endpoints but it doesn't work if no system is known after the rollback. Find a better solution!

Reply:
:+1: